app/src/main/res/values-et/about.xml

```diff
@@ -8,54 +8,54 @@
         <b>Autorid</b>
         <br/>
         m2049r, baltsar777, anhdres, keejef,
-        rehrar, EarlOfEgo, ErCiccione et al.
+        rehrar, EarlOfEgo, ErCiccione jt
```

Comment:
Is this the translation of 'et al'?

Reply:
`jt` is short for `ja teised` which means `and others` in Estonian.

Reply:
Ok, thanks for the explanation